app/models/item_order.rb

```diff
@@ -4,6 +4,8 @@ class ItemOrder <ApplicationRecord
   belongs_to :item
   belongs_to :order
 
+  enum status: %w(pending unfulfilled)
```

Comment:
Sheryl and I discussed possibly changing this from an enum to a boolean (like the `active?` column for items)